boards/arm/stm32f7/stm32f746g-disco/scripts/Make.defs

```diff
@@ -30,14 +30,14 @@ ifneq ($(CONFIG_DEBUG_NOOPT),y)
   ARCHOPTIMIZATION += $(MAXOPTIMIZATION) -fno-strict-aliasing
 endif
 
-ARCHCFLAGS = -fno-builtin
-ARCHCXXFLAGS = -fno-builtin -fno-exceptions -fcheck-new -fno-rtti
+ARCHCFLAGS = -fno-common -fno-builtin
+ARCHCXXFLAGS = -fno-common -fno-builtin -fno-exceptions -fcheck-new -fno-rtti
 
 ifeq ($(CONFIG_ARMV7M_TOOLCHAIN_CLANGL),y)
-  ARCHCFLAGS += -nostdlib -ffreestanding
-  ARCHCXXFLAGS += -nostdlib -ffreestanding
+  ARCHCFLAGS += -fno-common -nostdlib -ffreestanding
+  ARCHCXXFLAGS += -fno-common -nostdlib -ffreestanding
 else
-  ARCHCXXFLAGS += -fno-rtti
+  ARCHCXXFLAGS += -fno-common -fno-rtti
```

Comment:
It's strange that `-fno-rtti` is here conditionally and at line 34 unconditionally.

Reply:
Fixed